EIPS/eip-4750.md

```diff
@@ -96,25 +82,13 @@ If the code is valid EOF1, the following execution rules apply:
    
    Under `PC_post_instruction` we mean the PC position after the entire immediate argument of `CALLF`. Data stack height is saved as it was before function inputs were pushed.
    
-   *Note:* Code validation rules of [EIP-3670](./eip-3670.md) guarantee there is always an instruction following `CALLF` (since terminating instruction is required to be final one in the section), therefore `PC_post_instruction` always points to an instruction inside section bounds.
-   
+   *Note:* Code validation rules of [EIP-5450](./eip-5450.md) guarantee there is always an instruction following `CALLF` (since terminating instruction or unconditional jump is required to be final one in the section), therefore `PC_post_instruction` always points to an instruction inside section bounds.
 8. Sets `current_section_index` to `code_section_index` and `PC` to `0`, and execution continues in the called section.
 
-#### `JUMPF`
-
-1. Has one immediate argument,`code_section_index`, encoded as a 16-bit unsigned big-endian value.
-2. If data stack has less than `caller_stack_height + type[code_section_index].inputs` items, execution results in exceptional halt.
-3. If data stack has more than `caller_stack_height + type[code_section_index].inputs` items, discards the items between `caller_stack_height` and top `type[code_section_index].inputs` items, so that there are exactly `caller_stack_height +  type[code_section_index].inputs` items left.
-4. Charges 4 gas.
-5. Pops nothing and pushes nothing to data stack.
-6. Pushes nothing to return stack.
-7. Sets `current_section_index` to `code_section_index` and `PC` to `0`, and execution continues in the called section.
-
 #### `RETF`
 
 1. Does not have immediate arguments.
-2. If data stack has less than `caller_stack_height + type[code_section_index].outputs` items, execution results in exceptional halt.
-3. If data stack has more than `caller_stack_height + type[code_section_index].outputs` items, discards the items between `caller_stack_height` and top `type[code_section_index].outputs` items, so that there are exactly `caller_stack_height +  type[code_section_index].outputs` items left.
+2. If data stack does not equal `caller_stack_height + type[current_section_index].outputs` items, execution results in exceptional halt.
 4. Charges 4 gas.
```

Comment:
As stack cleanup is removed, restore previous gas cost of 3?

Reply:
```suggestion
4. Charges 3 gas.
```